### PR TITLE
Feature/login working

### PR DESCRIPTION
**Pull request by SamiUrias**
This is the login working. 
Only saves data into a variable ... 
This is my feature/login-fiexes merged with the branch of login with pino of @nandub 

For the moment is the only thing it do

### CODE REVIEW

File: server/src/controllers/auth/auth.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ exports.login = (req, res) => {
   console.log('--------------------------------------------------');
   console.log(req.body);
```
Comment (nandub) [inline server/src/controllers/auth/auth.js]: Please don't do this.  Use `.env` and change the `LOG_LEVEL` to debug you will see the whole request and response payload on your console.  I won't accept this kind of changes on our main repo.

File: server/src/controllers/users/user.js [hunk 2]
```diff
@@ -62,6 +62,7 @@ exports.get = (req, res) => {
  */
 exports.create = (req, res) => {
   req.log.debug("POST/create");
+  console.log('Request', req);
```
Comment (nandub) [inline server/src/controllers/users/user.js]: Please don't do this.  Use `.env` and change the `LOG_LEVEL` to debug you will see the whole request and response payload on your console.  I won't accept this kind of changes on our main repo.

File: server/server.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ import log from './src/logger';
 
 mongoose.Promise = global.Promise;
 
-const dbUrl = process.env.MONGODB_URI || process.env.MONGOLAB_URI || 'mongodb://localhost:27017/deardiiary';
+const dbUrl = process.env.MONGODB_URI || process.env.MONGOLAB_URI || 'mongodb://127.0.0.1:27017/deardiiary';
```
Comment (nandub) [inline server/server.js]: I'm good with it.

File: server/src/controllers/users/user.js (hunk 2 above)
Comment (SamiUrias) [reply server/src/controllers/users/user.js]: I remove this console log in the next commit

File: server/src/controllers/auth/auth.js (hunk 1 above)
Comment (SamiUrias) [reply server/src/controllers/auth/auth.js]: I remove this console.log in the next commit

**Pull request closed without merging**